packages/shared/src/components/notifications/NotificationsBell.tsx

```diff
@@ -37,12 +37,12 @@ function NotificationsBell({ compact }: { compact?: boolean }): ReactElement {
       <div
         className={classNames(
           'relative laptop:flex',
-          !isNewMobileLayout && 'hidden',
+          !isMobileLayout && 'hidden',
```

Comment:
Now this becomes very confusing. We negated the laptop then we negated the `isMobileLayout`, then we have a `laptop` CSS class.

We will need to merge the line directly above this.